frontend/src/metabase-lib/aggregation.ts

```diff
@@ -63,6 +63,9 @@ export function aggregationClause(
   operator: AggregationOperator,
   column?: ColumnMetadata,
 ): AggregationClause {
+  if (typeof column === "undefined") {
```

Comment:
There's a function `cljs.core/undefined?` that distinguishes JS `js/undefined` from `null`, even though both are treated as `nil?` by normal CLJS. This logic could go inside `metabase.lib.js/aggregation-clause` instead.

Reply:
It could, but is that preferable? And if it is, should the CLJS code have just a single arity (2) version?